catalogue/webapp/components/WorkCard/WorkCard.js

```diff
@@ -57,6 +57,13 @@ const Preview: ComponentType<SpaceComponentProps> = styled(Space).attrs(() => ({
   }
 `;
 
+// TODO: remove, hack to handle the fact that we are pulling through PDF thumbnails.
+// These will be removed from the API at some stage.
+function isPdfThumbnail(thumbnail): boolean {
+  // e.g. https://dlcs.io/iiif-img/wellcome/5/b28820769_WG_2006_PAAG-implementing-persistent-identifiers_EN.pdf/full/!200,200/0/default.jpg
+  return Boolean(thumbnail.url.match('.pdf/full'));
```

Comment:
I think the string '.pdf/full' is going to be converted to a regex `/abc.png/`, so should you escape the `.`?

Reply:
```JS
new RegExp('.pdf/full');
// => /.pdf\/full/
```

Reply:
ah, so the `/` is escaped, but not the `.` – is that what you want? I thought you want to check for literally '.pdf' rather than '{anyCharacter}pdf'?